test/ao/sparsity/test_qlinear_packed_params.py

```diff
@@ -154,8 +154,14 @@ def test_qlinear_packed_params(self, allow_non_zero_zero_points=False):
                     self.assertEqual(y1, y2)
 
 
+    def test_qlinear_packed_params_fbgemm(self):
```

Comment:
can we put @override_quantized_engine as a function decorator to reduce identation?

Reply:
oh sorry, I guess this is a contextmanager. I was talking about skipIfNoFBGEMM and skipIfNoQNNPACK, I guess we need to use both?

Reply:
Good point. I'll add the skip decorators.